### ISSUE TITLE
from step 5 on forward the the tutorial goes to poops. Very hard to follow for newbies

### ISSUE THREAD

**Issue by lixoten** [google-developer-training/basic-android-kotlin-compose-training-mars-photos#27]
**URL of codelab:**
https://developer.android.com/codelabs/basic-android-kotlin-compose-add-repository?continue=https%3A%2F%2Fdeveloper.android.com%2Fcourses%2Fpathways%2Fandroid-basics-compose-unit-5-pathway-2%23codelab-https%3A%2F%2Fdeveloper.android.com%2Fcodelabs%2Fbasic-android-kotlin-compose-add-repository#4

**Specify the language of the codelab if it is not English:**

**In which task and step of the codelab can this issue be found?**
5-10

**Describe the problem**
The logic in tutorial does not match the solution. Variables name differ, location of code is wrong.
Your developers need to do some QA, I am looking for work if you need help!!! LOL

**Steps to reproduce?**
1. Go to...
2. Click on...
3. See error...

**Versions**
_Android Studio version:_
_API version of the emulator:_

**Additional information**
_Include screenshots if they would be useful in clarifying the problem._

**Comment by wngu6** [comment]
At no point in the steps did it mention that the class `DefaultMarsPhotosRepository` has been renamed to `NetworkMarsPhotosRepository`, and that we change the function
```
override suspend fun getMarsPhoto(): List<MarsPhoto> {
    return marsApiService.getPhotos()
}
```
to
```
override suspend fun getMarsPhoto(): List<MarsPhoto> = marsApiService.getPhotos()
```

**Comment by osuleymanova** [comment]
Hello @lixoten,

Thanks for reaching out to us and sharing your feedback.
